Update Set-CsTeamsVoiceApplicationsPolicy.md (#10197)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

# Set-CsTeamsVoiceApplicationsPolicy

## SYNOPSIS

Modifies an existing Teams voice applications policy.

## SYNTAX

> ```ps

Reviewer: ```suggestion
```
```

You don't need to add powershell to the syntax section. Please check: https://learn.microsoft.com/PowerShell/scripting/community/contributing/powershell-style-guide

 [-AllowAutoAttendantHolidayRoutingChange <Boolean>] [-AllowCallQueueMusicOnHoldChange <Boolean>]
 [-AllowCallQueueOverflowSharedVoicemailGreetingChange <Boolean>]
 [-AllowCallQueueTimeoutSharedVoicemailGreetingChange <Boolean>]
 [-AllowCallQueueWelcomeGreetingChange <Boolean>] [-AllowCallQueueOptOutChange <Boolean>] [-AllowCallQueueAgentOptChange <Boolean>] [-AllowCallQueueMembershipChange <Boolean>] [-AllowCallQueueRoutingMethodChange <Boolean>] [-AllowCallQueuePresenceBasedRoutingChange <Boolean>]
 [-CallQueueAgentMonitorMode <Disabled | Monitor | Whisper | Barge | Takeover>]
 [-CallQueueAgentMonitorNotificationMode <Disabled | Agent>]
 [-AllowCallQueueLanguageChange <Boolean>] [-AllowCallQueueOverflowRoutingChange <Boolean>]
 [-AllowCallQueueTimeoutRoutingChange <Boolean>] [-AllowCallQueueNoAgentsRoutingChange <Boolean>]
 [-AllowCallQueueConferenceModeChange <Boolean>] [[-Identity] <String>]
 [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION

TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.

## EXAMPLES

### EXAMPLE 1

> ```ps

Reviewer 2: please whenever you intend to use powershell code block use powershell instead of ps

Accept wildcard characters: False
```

### -AllowAutoAttendantBusinessHoursChange

> When set to True users affected by the policy will be allowed to change the auto attendant's business hour schedule. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's business hour schedule.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change the auto attendant's business hours schedule. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's business hours schedule.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAutoAttendantTimeZoneChange

> When set to True users affected by the policy will be allowed to change the auto attendant's timezone. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's timezone.

Reviewer 4: time zone (instead of timezone)

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAutoAttendantLanguageChange

When set to True users affected by the policy will be allowed to change the auto attendant's language. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's language.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAutoAttendantHolidaysChange

When set to True users affected by the policy will be allowed to change the auto attendant's holiday schedules. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's holiday schedules.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAutoAttendantBusinessHoursRoutingChange

> When set to True users affected by the policy will be allowed to change the auto attendant's business hour call flow. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's business hour call flow.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change the auto attendant's business hours call flow. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's business hours call flow.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAutoAttendantAfterHoursRoutingChange

> When set to True users affected by the policy will be allowed to change the auto attendant's after-hour call flow. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's after-hour call flow.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change the auto attendant's after-hours call flow. When set to False (the default value) users affected by the policy will not be allowed to change the auto attendant's after-hours call flow.
```

Accept wildcard characters: False
```

### -AllowCallQueueOptOutChange

> When set to True users affected by the policy will be allowed to change the call queue whether to allow agent opt-out. When set to False (the default value) users affected by the policy will not be allowed to change the call queue whether to allow agent opt-out.

Reviewer 4: ... allowed to change the call queue opt-out setting that allows agents to opt-out of receiving calls.
.... will not be allowed to change the call queue opt-out setting that allows agents to opt-out of receiving calls.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueAgentOptChange

> When set to True users affected by the policy will be allowed to change the call queue's agent opt-in status. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's agent opt-in status.

Reviewer 4: ... allowed to change an agent's opt-in status in the call queue.  
... will not be allowed to chagne an agent's opt-in status in the call queue.  
Note: The call queue must be configured to allow agent's to opt-out in order for this option to work.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueMembershipChange

When set to True users affected by the policy will be allowed to change the call queue's users. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's users.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueRoutingMethodChange

When set to True users affected by the policy will be allowed to change the call queue's routing method. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's routing method.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueuePresenceBasedRoutingChange

> When set to True users affected by the policy will be allowed to change the call queue's presence aware routing. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's presence aware routing.

Reviewer 4: ... allowed to change the call queue's presence-based routing option.
... not allowed to change the call queue's presence-based routing option.

Accept wildcard characters: False
```

### -AllowCallQueueOptOutChange

> When set to True users affected by the policy will be allowed to change the call queue opt-out setting that allows agents to opt-out of receiving calls. When set to False (the default value) users affected by the policy will not be allowed to change the call queue opt-out setting.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change the call queue opt-out setting that allows agents to opt out of receiving calls. When set to False (the default value) users affected by the policy will not be allowed to change the call queue opt-out setting.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueAgentOptChange

> When set to True users affected by the policy will be allowed to change an agent's opt-in status in the call queue. When set to False (the default value) users affected by the policy will not be will not be allowed to change an agent's opt-in status in the call queue.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change an agent's opt-in status in the call queue. When set to False (the default value) users affected by the policy will not be allowed to change an agent's opt-in status in the call queue.
```

> Note: The call queue must be configured to allow agent's to opt-out in order for this option to work.

Reviewer 3: ```suggestion
Note that the call queue must be configured to allow agents to opt out in order for this option to work.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueMembershipChange

When set to True users affected by the policy will be allowed to change the call queue's users. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's users.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueueRoutingMethodChange

When set to True users affected by the policy will be allowed to change the call queue's routing method. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's routing method.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallQueuePresenceBasedRoutingChange

> When set to True users affected by the policy will be allowed to change the call queue's presence-based routing option. When set to False (the default value) users affected by the policy will not be not allowed to change the call queue's presence-based routing option.

Reviewer 3: ```suggestion
When set to True users affected by the policy will be allowed to change the call queue's presence-based routing option. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's presence-based routing option.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -CallQueueAgentMonitorMode

PARAMVALUE: Disabled | Monitor | Whisper | Barge | Takeover

When set to Disabled (the default value), users affected by the policy will not be allowed to monitor call sessions.

When set to Monitor, users affected by the policy will be allowed to monitor and listen to call sessions.

> When set to Whisper, users affected by the policy will be allowed to monitor call sessions and whisper to agent in the call.

Reviewer 3: ```suggestion
When set to Whisper, users affected by the policy will be allowed to monitor call sessions and whisper to an agent in the call.
```

> When set to Barge, users affected by the policy will be allowed to monitor call sessions, whisper to agent in the call, or join the call session.

Reviewer 3: ```suggestion
When set to Barge, users affected by the policy will be allowed to monitor call sessions, whisper to an agent in the call, or join the call session.
```

> When set to Takeover, users affected by the policy will be allowed to monitor call sessions, whisper to agent in the call, join the call session, or take over the call from agent.

Reviewer 3: ```suggestion
When set to Takeover, users affected by the policy will be allowed to monitor call sessions, whisper to an agent in the call, join the call session, or take over the call from an agent.
```
